### PR TITLE
Created Events for buttons

### PR DESCRIPTION
**Pull request by prius2055**
### Created Events for buttons. 

**In this branch, I have implemented the following;**

- Events for buttons
- Styling for the buttons
- imported calculate.js and operate.js
- Passed states and props as between child and parent components

### CODE REVIEW

File: src/components/Calculator.css [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+
+.calculator button {
+  border: 1px #777 solid;
+  padding: 50px;
+  font-weight: 700;
+  font-size: 18px;
+  cursor: pointer;
+
+  /* width: 125px; */
+}
```
Comment (AmaduKamara) [inline src/components/Calculator.css]: - [ ] Kindly remove commented unused code from your codebase as this makes your code not clean and professional. Please remove all used commented codes 👍🏽

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+  const [calculatorDisplay, setCalculatorDisplay] = useState({
+    next: 0,
+    total: null,
+    operation: null,
+  });
+
+  const displayHandler = (symbol) => {
+    setCalculatorDisplay(symbol);
+    // console.log(symbol);
+  };
```
Comment (AmaduKamara) [inline src/App.js]: - [ ] Kindly remove commented unused code from your codebase as this makes your code not clean and professional. Please remove all used commented codes 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @prius2055,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽 
✅ Used calculator logic provided 👍🏽
✅ Math logic functions work fine as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add a description of the project on your README. This is the Math Magician app for all maths lovers. You could write more in describing the project from the project requirement. Please work on that 👍🏽

Screenshot 2023-04-27 at 1 41 11 PM

- [ ] I'm sure you're working on the Math Magician project and not JavaScript Capstone. Kindly modify your README to have content related to this project 👍🏽

Screenshot 2023-04-27 at 1 41 41 PM

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by amejid — APPROVED** [review]
Hi @prius2055, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Calculator functioning as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**